main.nf

```diff
@@ -646,6 +647,67 @@ def d4_coverage_version(task) {
 	"""
 }
 
+process verifybamid2 {
+        cpus 16
+        memory '10 GB'
+        // publishDir "${OUTDIR}/contamination", mode: 'copy', overwrite: 'true', pattern: '*.selfSM'
+        tag "$id"
+        container = "${params.container_verifybamid2}"
+
+        input:
+                set group, id, file(bam), file(bai) from verifybamid2_bam
+
+        output:
+                file("${id}.result.selfSM")
+                file("${id}.result.Ancestry")
+                set group, file("*versions.yml") into ch_verifybamid2_versions
+
+        script:
+
+                if ( params.antype == "wgs") {
+                         """
+                         verifybamid2 \
+                         --SVDPrefix ${params.svdprefix} \
+                         --Reference $genome_file \
+                         --BamFile ${bam}
+
+                         mv result.selfSM ${id}.result.selfSM
+                         mv result.Ancestry ${id}.result.Ancestry
+                         ${verifybamid2_version(task)}
+                         """
+                }
+                else {
+                         """
+                         verifybamid2 \
+                         --DisableSanityCheck \
+                         --SVDPrefix ${params.svdprefix} \
+                         --Reference $genome_file \
+                         --BamFile ${bam}
+
+                         mv result.selfSM ${id}.result.selfSM
+                         mv result.Ancestry ${id}.result.Ancestry
+                         ${verifybamid2_version(task)}
+                         """
+                }
+
+
+        stub:
+                """
+                touch "${id}.result.selfSM"
+                touch "${id}.result.Ancestry"
+
+                ${verifybamid2_version(task)}
+                """
+}
+def verifybamid2_version(task) {
+        """
+        cat <<-END_VERSIONS > ${task.process}_versions.yml
+        ${task.process}:
+            VerifyBamID2: \$( echo \$( verifybamid2 --help 2>&1 | grep Version ) | sed "s/^.*Version://" )
```

Comment:
Have you verified that this looks correct in the `version.yaml` output?

Reply:
Here is the document with all versions. This version should be included here as well:

https://github.com/Clinical-Genomics-Lund/nextflow_wgs/blob/master/docs/list_of_all_used_software.md

Reply:
Yes looks good. I will add it to the list of versions asap,

Reply:
(Unresolved, so you can resolve when actually resolved ...)